### ISSUE TITLE
Word2Vec TSNEStandardExample gives Error at each iteration

### ISSUE THREAD

**Issue by mustafasari** [deeplearning4j/deeplearning4j#4572]
https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/nlp/tsne/TSNEStandardExample.java
Its dependency:     
```
<dependency>
            <groupId>org.deeplearning4j</groupId>
            <artifactId>deeplearning4j-ui_2.10</artifactId>
            <version>0.9.1</version>
</dependency>
```

Im going to try this example with words.txt but I gives the following error at the line of
->tsne.fit(weights);

3:40:05.995 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Calculating probabilities of data similarities...
13:40:05.995 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Handled 0 records
13:40:11.847 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 0 is -1722017.4903733607
13:40:12.446 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 1 is -1722010.0044403807
13:40:13.131 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 2 is -1722003.57197166
13:40:13.792 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 3 is -1721997.5659493157
13:40:14.456 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 4 is -1721991.7531823912
13:40:15.059 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 5 is -1721986.032710201
13:40:15.657 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 6 is -1721980.3574987103
13:40:16.403 [main] INFO org.deeplearning4j.plot.BarnesHutTsne - Error at iteration 7 is -1721974.7048414794

**Comment by mustafasari** [comment]
but i have used 
https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/src/main/resources/words.txt
i didn't understand ?

**Comment by AlexDBlack** [comment]
"Error" as in analogous to a neural network's loss function value.
